app/[locale]/institute/sections/central-library/collection-and-resources/page.tsx

```diff
@@ -0,0 +1,389 @@
+import Link from 'next/link';
+import { FaExternalLinkAlt } from 'react-icons/fa';
+
+import { Button } from '~/components/buttons';
+import Heading from '~/components/heading';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '~/components/ui';
+import { getTranslations } from '~/i18n/translations';
+import { cn } from '~/lib/utils';
+
+export default async function CollectionAndResources({
+  params: { locale },
+}: {
+  params: { locale: string };
+}) {
+  const EResourcesTableData = [
```

Comment:
Does this have to be static? If so, at least move it to another `.ts` file because it's so big